### ISSUE TITLE
Example to read Items

### ISSUE THREAD

**Issue by printetrain** [emrebekar/node-red-contrib-opcda-client#4]
Hello 
i am a newgie about node red and opc . I am searching for an examble to read an Item . I had installed two version of opc. I can read Items with the version node-red-contrib-opc-da but I can not read Items from  node-red-contrib-opcda-client. I Think that I  need the properties about the opcda read
![opc](https://user-images.githubusercontent.com/76214035/107143312-b45ad680-6934-11eb-91c5-f3bd4a7c47f0.JPG)

Thanks

**Comment by vaniiplon** [comment]
hi i am using matrikon simulation server. please help to connect with node opc da client to simulation server

**Comment by emrebekar** [comment]
Hi.

You can wire a trigger such as interval or anything else before read node.

7 Şub 2021 Pzr 13:08 tarihinde printetrain ***@***.***> şunu
yazdı:

**Comment by vaniiplon** [comment]
hi,
i had attached screenshot of configuration of opc da read node and error msgs i am getting
![opcda](https://user-images.githubusercontent.com/107458041/207827195-7b6d01ce-bd26-43cb-84cc-ef91476be559.JPG)
Please suggest what changes i need to do to connect to server

**Comment by emrebekar** [comment]
Have you applied the DCOM configuration?

15 Ara 2022 Per 12:48 tarihinde vnvasireddy ***@***.***> şunu
yazdı:

**Comment by vaniiplon** [comment]
Yes i had done with reference to Matrikon document. Please suggest if any reference document for DCOM settings

**Comment by WarmBoot** [comment]
> Yes i had done with reference to Matrikon document. Please suggest if any reference document for DCOM settings

@vnvasireddy I was getting that same error, but found a suggestion at [https://github.com/termux/termux-packages/issues/9266](url) that seems to have gotten me past that particular error.

I added the environment variable:

![image](https://github.com/emrebekar/node-red-contrib-opcda-client/assets/136015212/15889901-8609-4aa2-b103-f9f97133ebcb)

Unfortunately, I'm now getting the popular "access denied" error, but at least that seems a step closer...
